src/sentry/uptime/endpoints/uptime_ips.py

```diff
@@ -28,4 +17,5 @@ class UptimeIpsEndpoint(Endpoint):
     permission_classes = ()
 
     def get(self, _request: Request) -> HttpResponse:
-        return HttpResponse("\n".join(UPTIME_IP_ADDRESSES), content_type="text/plain")
+        ips: list[str] = list(options.get("uptime.uptime-ips-api-response"))
```

Comment:
was the corresponding option added somewhere else? worth adding a test?

Reply:
(or maybe there already is one)

Reply:
oh yeah need to update the test 👍